src/Integration.Vsix.UnitTests/CFamily/CLangAnalyzerTests.cs

```diff
@@ -96,21 +101,25 @@ public void ExecuteAnalysis_RequestCannotBeCreated_PCH_NoLogOutput()
 
             testSubject.ExecuteAnalysis("path", "charset", new[] { AnalysisLanguage.CFamily }, Mock.Of<IIssueConsumer>(), analysisOptions, CancellationToken.None);
 
+            testSubject.WaitForFactoryCallToComplete();
+
             testLogger.AssertNoOutputMessages();
         }
 
         [TestMethod]
         public void ExecuteAnalysis_RequestCanBeCreated_AnalysisIsTriggered()
         {
             var analysisOptions = new CFamilyAnalyzerOptions();
-            var requestFactory = CreateRequestFactory("path", analysisOptions, Mock.Of<IRequest>());
+            var request = CreateRequest("path");
```

Comment:
The request now needs to be valid i.e. have a non-null `RequestContext` property.